spec/features/users/user_logs_in_spec.rb

```diff
@@ -33,4 +33,13 @@
     expect(current_path).to eq(login_path)
     expect(page).to have_content("Invalid Credentials")
   end
+
+  scenario 'guest clicks on create account' do
+    visit login_path
```

Comment:
Great test, but it seems like it should live in the guest_creates_account_spec if you wouldn't mind taking a bit to review it.